monitor/dbt_project/models/alerts/alerts.sql

```diff
@@ -0,0 +1,15 @@
+{{
+  config(
+    materialized = 'incremental',
+    unique_key = 'alert_id',
+    merge_update_columns = ['alert_id'],
+    on_schema_change = 'append_new_columns'
+  )
+}}
+
+
+select *, false as alert_sent from {{ ref('elementary', 'failed_tests') }}
+
+{%- if is_incremental() %}
+    where {{ get_new_alerts_where_clause(this) }}
```

Comment:
This macro introduced an issue if the row_count of 'this' is 0 - it returns none